README.md

```diff
@@ -91,6 +91,19 @@ npm i
 npm run test:integration-local
 ````
 
+To generate  emailable allure report after running the integration test
+````
+ docker run --name allure -p 5050:5050  \
+                 -v ${PWD}/allure-results:/app/allure-results \
+                 -v ${PWD}/allure-reports:/app/default-report \
+                 frankescobar/allure-docker-service
+````
+
+To clean the allure results and allure reports folder 
+````
+npm run beforeIntTest
```

Comment:
The name of this implies it runs before integration tests, but as far as I can tell, it is only ever run manually.  Am I misunderstanding something here?

Reply:
Before running the integration tests, manually running this command 'npm run beforeIntTest' cleans the previously ran allure results and allure reports from the folders (allure-results and allure-reports)